Description: Adds an explicit dependency on JAX-WS since it's no longer provided by the JRE
Author: Emmanuel Bourg <ebourg@apache.org>
Bug: https://github.com/jmock-developers/jmock-library/pull/111

--- a/jmock/pom.xml
+++ b/jmock/pom.xml
@@ -53,6 +53,12 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.ws</groupId>
+            <artifactId>jaxws-api</artifactId>
+            <version>2.3.0</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
     <build>
